tests/generation/test_hub.py

```diff
@@ -66,18 +48,6 @@ def neuron_push_model_id(model_id):
     return repo_id
 
 
-@is_inferentia_test
-@requires_neuronx
-def test_push_decoder_to_hub():
-    model_id = "hf-internal-testing/tiny-random-gpt2"
-    model = NeuronModelForCausalLM.from_pretrained(model_id, export=True)
-    with TemporaryDirectory() as tmpdir:
-        model.save_pretrained(tmpdir)
-        ignore_patterns = [model.CHECKPOINT_DIR + "/*"]
-        neuron_push_decoder_id = neuron_push_model_id(model_id)
-        _test_push_to_hub(model, tmpdir, neuron_push_decoder_id, ignore_patterns)
-
-
 @is_inferentia_test
 @requires_neuronx
 def test_seq2seq_model_from_hub():
```

Comment:
Ok, I shall do the same for seq2seq tests then.

Reply:
Yes, but I was wondering if you wanted to keep these anyway.